# 15: Physical System Mapping

## Objetivo
Mapear o modelo de decoerência (N² vs N) para ordens de grandeza de sistemas experimentais reais: átomos de Rydberg, interferometria molecular e gases frios.

**Tag Epistemológica: PREVISÃO**

> [!IMPORTANT]
> Este artefato tem finalidade estrita de previsão paramétrica. Nenhuma das escalas aqui mapeadas constitui confirmação experimental do modelo.

In [1]:
import pandas as pd
import numpy as np

# Parâmetros calibrados na rodada anterior
a = 0.014
b = 0.00014

# Sistemas físicos e estimativas típicas
sistemas = [
    "Gases Frios (ex: BEC em armadilha)",
    "Interferometria Molecular (ex: C60, macro-moléculas)",
    "Átomos de Rydberg (fortes interações dipolo-dipolo)"
]

N_tipico = [
    10000, # Gases frios têm N grande, trataremos N como componentes interagentes (10^4)
    50,    # Interferometria molecular (N como número de átomos/modos)
    30     # Rydberg (N na bolha de bloqueio)
]

gamma_tipico_range = [
    "0.1 - 10",
    "1 - 100",
    "10^3 - 10^5"
]

gamma_linear = [a * N for N in N_tipico]
gamma_quadratico = [b * (N**2) for N in N_tipico]

df = pd.DataFrame({
    "Sistema Físico": sistemas,
    "N Típico (ordem)": N_tipico,
    "Gamma Experimental (s^-1)": gamma_tipico_range,
    "Gamma Previsto Linear": gamma_linear,
    "Gamma Previsto Quadrático": gamma_quadratico
})

df

,Sistema Físico,N Típico (ordem),Gamma Experimental (s^-1),Gamma Previsto Linear,Gamma Previsto Quadrático
0,Gases Frios (ex: BEC em armadilha),10000,0.1 - 10,140.00,14000.000
1,"Interferometria Molecular (ex: C60, macro-molé...",50,1 - 100,0.70,0.350
2,Átomos de Rydberg (fortes interações dipolo-di...,30,10^3 - 10^5,0.42,0.126


## Conclusão do Mapeamento

1. **Gases Frios**: Para N=10000, o modelo quadrático prevê uma taxa de decoerência altíssima (14000 s^-1), o que indica que, em sistemas de alto N, o limite N² rapidamente destrói o contraste, exceto para tempos de observação muito curtos.
2. **Interferometria Molecular**: N=50 cai em uma faixa onde os modelos divergem sutilmente. A taxa quadrática prevista (~0.35 s^-1) é experimentalmente factível e comparável à linear.
3. **Átomos de Rydberg**: Sistemas de forte interação e alta decoerência. A predição do modelo está alinhada qualitativamente, mas a dependência na escala física exigiria recalibrar (a, b) especificamente para estados de Rydberg.

**Existem sistemas plausíveis?** Sim, especialmente moléculas grandes (N~50) e micro-nuvens de átomos.